src/index.ts

```diff
@@ -46,8 +46,10 @@ export function main(port: number): Server {
     require('probot/lib/apps/sentry'),
     require('probot/lib/apps/stats'),
   ]
-
-  process.on('unhandledRejection', probot.errorHandler)
+  ;(process as NodeJS.EventEmitter).on(
```

Comment:
Evil! ;-) Can that be avoided with a different syntax?

The reason why I prefer having semi-columns.

Reply:
It's super ugly, I agree, but sadly that's the only way right now. We should remove it once the TS team resolves it.